[Windows] Compiling from sources issue

I've followed the manual : https://deno.land/manual.html#buildfromsource

And i've the error:
```
 GetWinSDKDir
    import vs_toolchain
ImportError: No module named vs_toolchain
```
After checking in https://github.com/denoland/deno/blob/master/.appveyor.yml i have updated my PYTHONPATH to `C:\temp\deno\third_party\python_packages`
then i got:
`Exception: Visual Studio version 2015 (from GYP_MSVS_VERSION) not supported. Supported versions are: 2017, 2019`
Problem is i have already installed nodejs before and it sets this env variable. After changing it to :
`GYP_MSVS_VERSION=2017`
It works!
Maybe the documentation needs to be updated or i got a weird configuration?

Another issue now is build.py which gives me : 
```
[176/552] COPY ../../third_party/flatbuffers/js/flatbuffers.js gen/flatbuffers.js
FAILED: gen/flatbuffers.js
C:/python27/python.exe ../../build/toolchain/win/tool_wrapper.py recursive-mirror ../../third_party/flatbuffers/js/flatbuffers.js gen/flatbuffers.js
Traceback (most recent call last):
  File "../../build/toolchain/win/tool_wrapper.py", line 29, in <module>
    import win32file    # pylint: disable=import-error
ImportError: DLL load failed: %1 nÆest pas une application Win32 valide.
[181/552] CXX obj/third_party/v8/torque_base/torque_base_jumbo_1.obj
ninja: build stopped: subcommand failed.
```

I've tried to tweak my env variables but can't make it work. Any idea?

--- a/website/manual.md
+++ b/website/manual.md
@@ -123,9 +123,20 @@ $ deno run https://deno.land/welcome.ts
 
 ### Build from source
 
+Clone on Unix/MacOs
+
 ```bash
 # Fetch deps.
 git clone --recurse-submodules https://github.com/denoland/deno.git
+```
+
+Clone on Windows (with Admin privileges)
+
+```bash
+git clone -c core.symlinks=true --recurse-submodules https://github.com/denoland/deno.git
+```
+
+```bash
 cd deno
 ./tools/setup.py
 

